src/components/Text/Text.tsx

```diff
@@ -33,28 +33,33 @@ interface TextProps
     inverted?: boolean;
     /**
      * Adjust color to indicate secondary information
+     * @deprecated
      */
     weak?: boolean;
+    /**
+     * Adjust color to indicate secondary information
+     */
+    secondary?: boolean;
     /**
      * Adjust color to display a disabled text element
      */
     disabled?: boolean;
 }
 
-function determineTextColor({ weak, inverted, disabled }: TextProps) {
+function determineTextColor({ weak, secondary, inverted, disabled }: TextProps) {
     if (disabled) {
-        return inverted ? Colors.AUTHENTIC_BLUE_550 : Colors.AUTHENTIC_BLUE_350;
+        return inverted ? SemanticColors.text.disabledInverted : SemanticColors.text.disabled;
     }
 
-    if (weak) {
-        return inverted ? Colors.AUTHENTIC_BLUE_350 : Colors.AUTHENTIC_BLUE_550;
+    if (secondary || weak) {
```

Comment:
We have an amazing helper function that sends a console message in the dev environment to let the developers know about the deprecation: https://github.com/freenowtech/wave/blob/main/src/utils/deprecatedProperty.ts. I would suggest we use it here as well, when the `weak` prop is set. It will make upgrading easier and encourage teams to use the new `secondary` prop instead.

Reply:
Thanks for reminding me about that function 👍